cdap-docs/developer-manual/source/building-blocks/services.rst

```diff
@@ -234,22 +222,4 @@ Both the number of service threads and the thread keep-alive time can be altered
 
 Services Examples
 =================
-- The :ref:`Purchase example <examples-purchase>` includes two services, ``CatalogLookupService``
-  and ``PurchaseHistoryService``; the latter **retrieves a specified customer's purchase 
-  history in a JSON format from a dataset.**
-
-- The :ref:`Spark example <examples-spark-k-means>` includes a service that **responds with
-  a calculated center from a dataset based on an index parameter.**
-
-- For another example of **a service reading from a dataset,** see the :ref:`Spark PageRank
-  example <examples-spark-page-rank>`.
-
-- For an example of **using path and query parameters,** see the 
-  :ref:`WordCount example <examples-word-count>`, where the class ``RetrieveCountsHandler``
-  retrieves a variety of statistics from datasets depending on the path supplied. 
-
-- Almost all of the :ref:`how-to guides <guides-index>` demonstrate the use of services.
```

Comment:
aren't we keeping the guides?

Reply:
Yes, but this statement is no longer true. A majority of them don't demonstrate services.